node/cmd/sentinel/main.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"strings"
+	"sync"
+
+	"bisonai.com/orakl/node/pkg/checker/balance"
+	"bisonai.com/orakl/node/pkg/checker/dal"
+	"bisonai.com/orakl/node/pkg/checker/event"
+	"bisonai.com/orakl/node/pkg/checker/health"
+	"bisonai.com/orakl/node/pkg/checker/peers"
+	"bisonai.com/orakl/node/pkg/checker/signer"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
+
+func main() {
+	ctx := context.Background()
+	logLevel := os.Getenv("LOG_LEVEL")
+	if logLevel == "" {
+		logLevel = "info"
+	}
+	zerolog.SetGlobalLevel(getLogLevel(logLevel))
```

Comment:
sentinel can use logscribe now, should we integrate it in this PR or a separate one? 😄

Reply:
let's fully focus on just moving the codebase here 😅 integration can be done after